modules/machine-user-infra-machines-ibm-z-kvm.adoc

```diff
@@ -16,6 +16,31 @@ You can create more {op-system-first} compute machines for your cluster by using
 * An HTTP or HTTPS server is set up.
 
 .Procedure
+// Step 1 is a workaround for https://issues.redhat.com/browse/OCPBUGS-18394
+// Can be removed when bug is fixed.
+. Disable UDP aggregation
++
+Currently, UDP aggregation is not supported on {ibmzProductName} and is not automatically deactivated on multi-architecture compute clusters with an `x86_64` control plane with additional `s390x` compute machines. To ensure that the addtional compute nodes are added to the cluster correctly, you must manually disable UDP aggregation.
```

Comment:
To clarify what has the additional machines. Probably not needed for experts, but might help translators.
```suggestion
Currently, UDP aggregation is not supported on {ibmzProductName} and is not automatically deactivated on multi-architecture compute clusters with an `x86_64` control plane and additional `s390x` compute machines. To ensure that the addtional compute nodes are added to the cluster correctly, you must manually disable UDP aggregation.
```